### PR TITLE
Dblatcher/signup confirmation wording change

### PR DESCRIPTION
**Pull request by dblatcher**
## What does this change?
The all-newsletters page uses the same confirmation text as the embed forms, being the values for `emailListing.emailEmbed.successHeadline` and `emailListing.emailEmbed.successDescription` from the newsletters api.

Those values will need to be updated to co-ordinate with a forthcoming change to the 'sign-up journey' (user won't need to verify their subscriptions after signing up), so we don't want to merge this change until after the 'sign-up journey' change has gone line and after updating the API values.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### before: 

Screenshot 2022-02-28 at 15 46 48

### after: 
Screenshot 2022-02-28 at 15 45 55

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: applications/app/views/signup/newsletterContent.scala.html [hunk 1]
```diff
@@ -36,9 +36,12 @@
 
 
                 <div class="signup-confirmation is-hidden js-signup-confirmation">
-                    @fragments.inlineSvg("envelope", "icon")
-                    <div class="signup-confirmation__message">
-                        <h3 class="signup-confirmation--success">Check your Inbox!</h3>
+                    <div class="signup-confirmation__success-headline-container">
+                        @fragments.inlineSvg("envelope", "icon")
+                        <span class="signup-confirmation--success">@Html(emailListing.emailEmbed.successHeadline)</span>
```
Comment (oliverlloyd) [inline applications/app/views/signup/newsletterContent.scala.html:41]: Is there a way to keep the same styling and use of tags as before? I haven't tested this but my guess is the `h3` will be better for accessibility.

Also, it seems we've changed where the background grey sits which has the effect of making the confirmation message look a bit buttony, which might entice people to try and click on it.

## Before
Screenshot 2022-02-28 at 15 02 09

## After
Screenshot 2022-02-28 at 15 02 29

File: static/src/stylesheets/module/signup/_newsletters.scss [hunk 2]
```diff
@@ -281,22 +281,24 @@
...
 
     .signup-confirmation--success {
         @include fs-textSans(3);
+        font-weight: 700;
         color: $brightness-7;
         display: inline;
     }
 
     .inline-icon {
-        background-color: $brightness-86;
```
Comment (oliverlloyd) [inline static/src/stylesheets/module/signup/_newsletters.scss:299]: I'm guessing this is where the buttoness crept in

**Review by oliverlloyd — COMMENTED** [review]
Hey David, Sorry to be a nag but could we make the new message less like a button, please? It looks very inviting to click on, thanks!

File: applications/app/views/signup/newsletterContent.scala.html (hunk 1 above)
Comment (dblatcher) [reply applications/app/views/signup/newsletterContent.scala.html:41]: Hi Oliver - thanks, that's a fair point. The mark-up and styling does need to be somewhat different from before as there are two lines of confirmation text now, but I'm sure I can change the appearance of the first to be less button-y and use a h3.

Comment (dblatcher) [reply applications/app/views/signup/newsletterContent.scala.html]: That's done - I've added new screenshot

**Review by oliverlloyd — APPROVED** [review]
Thanks for making those changes, looks great

**Pull request merged**